packages/swingset-liveslots/src/liveslots.js

```diff
@@ -1499,7 +1499,8 @@ function build(
   async function bringOutYourDead() {
     await scanForDeadObjects();
     // now flush all the vatstore changes (deletions) we made
-    vom.flushStateCache();
+    // eslint-disable-next-line no-use-before-define
+    afterDispatchActions();
```

Comment:
Won't the normal dispatch flow call `afterDispatchActions` also?

Reply:
I thought so too, but our `dispatch` treats BOYD and `stopVat` specially, and doesn't route them through the same code path, so they don't get an `afterDispatchActions`:

```js
  async function dispatch(delivery) {
    if (delivery[0] === 'bringOutYourDead') {
      return meterControl.runWithoutMeteringAsync(bringOutYourDead);
    } else if (delivery[0] === 'stopVat') {
      return meterControl.runWithoutMeteringAsync(() => stopVat(delivery[1]));
    } else {
      let complete = false;
      const p = Promise.resolve(delivery).then(unmeteredDispatch);
      p.finally(() => (complete = true));
      return gcTools.waitUntilQuiescent().then(() => {
        afterDispatchActions();
        return complete ? p : Promise.reject('buildRootObject unresolved');
      });
    }
  }
```

The two code paths are sufficiently different that I don't see an obvious way to merge them.

I'll add a comment explaining why it's necessary.